<launch>
	<!-- IMU NODE -->
	<!-- <node
		pkg="barc"
		type="imu_data_acquisition.py"
		name="imu_node" >
		<param
			name="port"
			value="/dev/ttyACM0" />
	</node> -->

	<!-- ARDUINO NODE -->
    <!--  * encoders and ultrasound sensors -->
	<node
		pkg="rosserial_python"
		type="serial_node.py"
		name="arduino_node" >
		<param
			name="port"
			value="/dev/ttyUSB1" />
	</node>

	<!-- global estimation node -->
	<!-- <node
		pkg="barc"
		type="global_estimation_XY.py"
		name="position_node" />
	</node> -->

	<!-- OPEN LOOP MANUEVERS -->
	<node
		pkg="barc"
		type="controller_OL_mpc.py"
		name="auto_mode" >
   </node>

    <!-- Record the experiment data -->

</launch>
